### PR TITLE
adding FakePerth

### PR DESCRIPTION
**Pull request by 1ucian0**
This PRs adds FakePerth based on `ibm_perth`.

### CODE REVIEW

File: releasenotes/notes/ibm_perth-5b1e9308dc302e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The fake backend :class:`~FakePerth` was added to with the information from IBM Quantum `ibm_perth` system.
```
Comment (1ucian0) [inline releasenotes/notes/ibm_perth-5b1e9308dc302e2e.yaml:4]:
```suggestion
    The fake backend :class:`~FakePerth` was added with the information from IBM Quantum `ibm_perth` system.
```

File: qiskit/providers/fake_provider/backends/perth/__init__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Mock perth backend"""
```
Comment (HuangJunye) [inline qiskit/providers/fake_provider/backends/perth/__init__.py:13]: Just nitpicking. Since we move the modules from `qiskit.test.mock` to `qiskit.providers.fake_provider`, can we also get rid of "mock" everywhere? This is exactly the same as other current fake backends. We can either make this changes for all the new ones, or we keep the old doc string and make a separate PR to correct them all. What do you think is better?

```suggestion
"""Fake Perth device (7 qubit)."""
```

**Review by HuangJunye — COMMENTED** [review]
Overall LGTM. Just one small comment.

File: qiskit/providers/fake_provider/backends/perth/__init__.py (hunk 2 above)
Comment (1ucian0) [reply qiskit/providers/fake_provider/backends/perth/__init__.py]: done

File: qiskit/providers/fake_provider/backends/perth/fake_perth.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+"""
+Fake Perth device (7 qubit).
+"""
+
+import os
+from qiskit.providers.fake_provider import fake_backend
+
+
+class FakePerthV2(fake_backend.FakeBackendV2):
```
Comment (mtreinish) [inline qiskit/providers/fake_provider/backends/perth/fake_perth.py:22]: If there is no V1 variant we don't need to mark this as V2

```suggestion
class FakePerth(fake_backend.FakeBackendV2):
```

Comment (1ucian0) [reply qiskit/providers/fake_provider/backends/perth/fake_perth.py]: sounds good. So, the type is the way to check the version?

Comment (mtreinish) [reply qiskit/providers/fake_provider/backends/perth/fake_perth.py]: You can either type check, or there is a `.version` class attribute set in the base abstract class. So you can do something like: `backend.version >= 2` or something like that.

**Review by mtreinish — APPROVED** [review]
LGTM, thanks for the updates

**Comment by 1ucian0** [comment]
Awesome. Ill update the rest.
